x-pack/plugins/case/common/api/connectors/servicenow_itsm.ts

```diff
@@ -7,10 +7,13 @@
 
 import * as rt from 'io-ts';
 
+// New fields should also be added at: x-pack/plugins/case/server/connectors/case/schema.ts
 export const ServiceNowITSMFieldsRT = rt.type({
   impact: rt.union([rt.string, rt.null]),
   severity: rt.union([rt.string, rt.null]),
   urgency: rt.union([rt.string, rt.null]),
+  category: rt.union([rt.string, rt.null]),
+  subcategory: rt.union([rt.string, rt.null]),
```

Comment:
the [issue](https://github.com/elastic/security-team/issues/477) says to add `incidentType`, confused about `category` and `subcategory`. Did the requirements change?

Reply:
No, ServiceNow does not have `incidentType`. The equivalent is `category` and `subcategory`.